.vib/schema-registry/cypress/cypress/integration/schema_registry_spec.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="cypress" />
+import body from '../fixtures/schema.json';
+
+it('can access the API and obtain global compatibility level', () => {
+  cy.request({
+    method: 'GET',
+    url: '/config',
+    form: true,
+  }).then((response) => {
+    expect(response.status).to.eq(200);
+    expect(response.body.compatibilityLevel).to.contain('BACKWARD');
```

Comment:
I would as `BACKWARD` as a Cypress.env given it is a value configurable in the chart and used in the test. This is a similar scenario to what we do in Goss.

Reply:
Done 👍 
At first I thought you were referring to `cy.fixtures`, but then I replaced it with `Cypress.env` as suggested.